ngraph/core/include/ngraph/op/prior_box.hpp

```diff
@@ -44,7 +44,7 @@ namespace ngraph
             float step = 1.0f;
```

Comment:
According to spec (https://docs.openvinotoolkit.org/latest/openvino_docs_ops_detection_PriorBox_1.html) step shall be initialized with 0.